bin/bails-menu

```diff
@@ -33,24 +33,26 @@ export DATA_DIR="/live/persistence/TailsData_unlocked/Persistent/.bitcoin"
 export BAILS_DIR="$LOCAL_DIR"
 
 onlynet_onion() {
-zenity --question --title='Network Privacy' --width=420 --text='Bails offers different settings for connecting to the Bitcoin network. By default, Bails connects to both clearnet and Tor (onion) peers, providing a balance between usability and privacy.
+    zenity --question --title='Network Privacy' --width=420 --text='Bails offers different settings for connecting to the Bitcoin network. By default, Bails connects to both clearnet and Tor (onion) peers, providing a balance between usability and privacy.
```

Comment:
We need to edit this dialog text to say "By default, Bitcoin Core connects"

Bails doesn't connect to anything except those wgets thankfully and to the github if you ask it to update itself.

Reply:
Will do.

Reply:
Resolved with 0825354d497ee1b501c19063142a88a74b172bd4.